lizrd/scripts/grid.py

```diff
@@ -52,6 +53,8 @@
 COPIED_CODE_PATH = ""
 if __name__ == "__main__":
     runner = get_machine_backend()
+    CACHE_PATH = os.getenv("CACHE_PATH", get_cache_path(runner))
+    HF_DATASETS_CACHE = os.getenv("HF_DATASETS_CACHE", CACHE_PATH)
```

Comment:
Why are we splitting this into two different things, i.e. CACHE_PATH and HF_DATASETS_CACHE?

Reply:
Make sense